src/Compilers/Core/Portable/CommandLine/AnalyzerConfig.cs

```diff
@@ -249,4 +255,20 @@ public Section(string name, ImmutableDictionary<string, string> properties)
             public ImmutableDictionary<string, string> Properties { get; }
         }
     }
+
+    internal sealed class GlobalAnalyzerConfig
+    {
+        internal AnalyzerConfig.Section GlobalSection { get; }
+
+        internal ImmutableArray<AnalyzerConfig.Section> NamedSections { get; }
+
+        internal const string ConfigPath = "<Global Config>";
```

Comment:
This is only ever used as a diagnostic argument in an `AnalyzerConfigSet`. I would consider moving this constant to this type as it's only relevant there. Adding it here, to me at least, implies that it has a broader use case.